src/Locale/pl_PL/default.po

```diff
@@ -0,0 +1,3106 @@
+
+# January
+msgid "January"
+msgstr "Styczeń"
+
+# February
+msgid "February"
+msgstr "Luty"
+
+# Mar
+msgid "March"
+msgstr "Marzec"
+
+# April
+msgid "April"
+msgstr "Kwiecień"
+
+# May
+msgid "May"
+msgstr "Maj"
+
+# June
+msgid "June"
+msgstr "Czerwiec"
+
+# July
+msgid "July"
+msgstr "Lipiec"
+
+# August
+msgid "August"
+msgstr "Sierpień"
+
+# September
+msgid "September"
+msgstr "Wrzesień"
+
+# October
+msgid "October"
+msgstr "Październik"
+
+# November
+msgid "November"
+msgstr "Listopad"
+
+# December
+msgid "December"
+msgstr "Grudzień"
+
+# Jan
+msgid "JanuaryShort"
+msgstr "Sty"
+
+# Feb
+msgid "FebruaryShort"
+msgstr "Lut"
+
+# Mar
+msgid "MarchShort"
+msgstr "Mar"
+
+# Apr
+msgid "AprilShort"
+msgstr "Kwi"
+
+# May
+msgid "MayShort"
+msgstr "Maj"
+
+# Jun
+msgid "JuneShort"
+msgstr "Cze"
+
+# Jul
+msgid "JulyShort"
+msgstr "Lip"
+
+# Aug
+msgid "AugustShort"
+msgstr "Sie"
+
+# Sep
+msgid "SeptemberShort"
+msgstr "Wrz"
+
+# Oct
+msgid "OctoberShort"
+msgstr "Paź"
+
+# Nov
+msgid "NovemberShort"
+msgstr "Lis"
+
+# Dec
+msgid "DecemberShort"
+msgstr "Gru"
+
+# Monday
+msgid "Monday"
+msgstr "Poniedziałek"
+
+# Tuesday
+msgid "Tuesday"
+msgstr "Wtorek"
+
+# Wednesday
+msgid "Wednesday"
+msgstr "Środa"
+
+# Thursday
+msgid "Thursday"
+msgstr "Czwartek"
+
+# Friday
+msgid "Friday"
+msgstr "Piątek"
+
+# Saturday
+msgid "Saturday"
+msgstr "Sobota"
+
+# Sunday
+msgid "Sunday"
+msgstr "Niedziela"
+
+# Mo
+msgid "MondayShort"
+msgstr "Pn"
+
+# Tue
+msgid "TuesdayShort"
+msgstr "Wt"
+
+# Wed
+msgid "WednesdayShort"
+msgstr "Śr"
+
+# Thu
+msgid "ThursdayShort"
+msgstr "Cz"
+
+# Fri
+msgid "FridayShort"
+msgstr "Pt"
+
+# Sat
+msgid "SaturdayShort"
+msgstr "So"
+
+# Sun
+msgid "SundayShort"
+msgstr "Nd"
+
+# Show more
+msgid "Show_more"
+msgstr "Pokaż więcej"
+
+# Show less
+msgid "Show_less"
+msgstr "Pokaż mniej"
+
+# Close
+msgid "Close"
+msgstr "Zamknij"
+
+# cart
+msgid "route_cart"
+msgstr "cart"
+
+# Remove products from cart
+msgid "Empty_cart"
+msgstr "Usuń produkty z koszyka"
+
+# deposit
+msgid "deposit"
+msgstr "kaucja"
+
+# Really empty cart?
+msgid "Really_empty_cart?"
+msgstr "Na pewno opróżnić koszyk?"
+
+# Load past order
+msgid "Load_past_order"
+msgstr "Wczytaj poprzednie zamówienie"
+
+# <p>The selected order will be loaded and <b>the current cart will be emptied</b>.</p><p>You can add products afterwards.</p>
+msgid "Load_past_order_dialog_description_html"
+msgstr "<p>Wybrane zamówienie zostanie wczytane. <b>Bieżąca zawartość koszyka zostanie usunięta</b>.</p><p>Będzie można dodać produkty ponownie.</p>"
+
+# Remove from cart?
+msgid "Remove_from_cart?"
+msgstr "Usunąć z koszyka?"
+
+# each
+msgid "for_each"
+msgstr "każdy"
+
+# approx.
+msgid "approx."
+msgstr "około"
+
+# Home
+msgid "Home"
+msgstr "Główna"
+
+# manufacturers
+msgid "route_manufacturer_list"
+msgstr "manufacturers"
+
+# Manufacturers
+msgid "Manufacturers"
+msgstr "Producenci"
+
+# news
+msgid "route_news_list"
```

Comment:
msgids starting with "route_" are routes (visible in url) and should be translated too
eg. www.yourdomain.com/news

Reply:
I know. I have translated routes initially but the page wasn't working properly so I thought it was because of translated routes and reverted them back to english.
It didn't help tho and page still works weird.

Reply:
ok I'll have a look on it later